kolibri/core/assets/src/utils/appCapabilities.js

```diff
@@ -0,0 +1,16 @@
+import urls from 'kolibri.urls';
+import plugin_data from 'plugin_data';
+
+const appCapabilities = plugin_data.appCapabilities || {};
+
+export const can = {
+  launchIntent: appCapabilities.launch_intent,
+};
+
+export const actionUrls = {
+  get launchIntent() {
+    return urls[`kolibri:kolibri.plugins.app:appcommands-launch_intent`]
+      ? urls[`kolibri:kolibri.plugins.app:appcommands-launch_intent`]()
+      : null;
+  },
+};
```

Comment:
This is fine for now, but I don't know that `actionUrls` is quite the interface I would have expected `appCapabilities` to expose. I might have expected more of a set of Vuex actions or just plain functions that get called. For example, maybe something like:



```js
export const capabilityHandlers = {
  shareFile(filename, message) {
    const url = urls[`kolibri:kolibri.plugins.app:appcommands-share_file`];
    if (!url) {
      logger.warn("not implemented };
      return;
    }
    return client.post({
      path: url,
      method: 'POST',
      params: { filename, message },
    });
};
```

Reply:
Yeah, exposing functions that then handle things internally seems like a nice, clean interface (minimal surface area, for easy migration paths).